CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**3.3.0 - 03/17/25**
```

Comment:
The current implementation will not be backwards compatible

Reply:
I feel like this might actually just be minor? It's a weird one, though, b/c it's a bugfix which is tytpically patch, but it's not backwards compatible which is typically major. Minor is usually reserved for backwards compatible new features (this isn't a new feature). Let's see what Rajan says.

Reply:
Major release